USAGE.md

```diff
@@ -120,7 +120,7 @@ For guidance on what CSL JSON should be like for different document types, refer
 ## Manuscript metadata
 
 [`content/metadata.yaml`](content/metadata.yaml) contains manuscript metadata that gets passed through to Pandoc, via a [`yaml_metadata_block`](http://pandoc.org/MANUAL.html#extension-yaml_metadata_block).
-`metadata.yaml` should contain the manuscript `title`, `authors` list, and `keywords`.
+`metadata.yaml` should contain the manuscript `title`, `authors` list, `keywords`, and language (`lang`).
```

Comment:
@dhimmel can we link to a list of valid language codes?

Reply:
From [Pandoc docs](http://pandoc.org/MANUAL.html#language-variables):

> `lang`: identifies the main language of the document, using a code according to [BCP 47](https://tools.ietf.org/html/bcp47) (e.g. `en` or `en-GB`). For some output formats, pandoc will convert it to an appropriate format stored in the additional variables `babel-lang`, `polyglossia-lang` (LaTeX) and `context-lang` (ConTeXt).

I will look for a BCP 47 list of language codes.

Reply:
BCP 47 codes can combine a language and region. Language codes are [defined here](http://www.loc.gov/standards/iso639-2/php/code_list.php). Region codes [are here](https://en.wikipedia.org/wiki/ISO_3166-1).

These BCP 47 codes can get quite complex [it seems](https://www.w3.org/International/articles/language-tags/) as they can have the following parts:

> language-extlang-script-region-variant-extension-privateuse

There's an unofficial validator at http://schneegans.de/lv/ that may be helpful. Wonder how complicated we want to make these docs!

Reply:
Great info.  Linking to one or two of those resources could help users without adding much more complexity to these manubot docs.

Reply:
Fix proposed in https://github.com/greenelab/manubot-rootstock/pull/83